src/quo2/foundations/colors.cljs

```diff
@@ -39,6 +39,14 @@
         (alpha light-color light-opacity)
         (alpha dark-color dark-opacity))))))
 
+(defn theme-color-with-blur
```

Comment:
yep, I've just seen across multiple components where we have 4 variations of colors (light, dark, light +blur, dark + blur).
It's pretty common across the designs really, see "src/quo2/components/selectors/styles.cljs" and it could use this function there. 

Re: "AFAIK, we don't dinamically change from blur colors to non-blur colors as we do with dark and light colors."
Not sure what you mean by - dynamically changing color sets? 

most of the blur variations have their own colors than their respective light or dark non blur implementation.

Reply:
but yep, happy to adjust this helper and also localize it to this style file if other devs see this to be the case too 👍

Reply:
> Not sure what you mean by - dynamically changing color sets?

I mean, a component using `(theme-colors light-color dark-color)`  will return `light-color` or `dark-color` depending on user preferences (dinamically).

But we won't swap between a `light-color` and a `blur-light-color`, I mean, a component using a blur color will always use a blur color, and it'll swap  between `blur-light-color` and `blur-dark-color` which is already covered by the `theme-colors` fn.